src/Open3D/Geometry/HalfEdgeTriangleMesh.cpp

```diff
@@ -316,9 +326,13 @@ HalfEdgeTriangleMesh &HalfEdgeTriangleMesh::Transform(
 }
 
 HalfEdgeTriangleMesh &HalfEdgeTriangleMesh::Translate(
-        const Eigen::Vector3d &translation) {
+        const Eigen::Vector3d &translation, bool relative) {
+    Eigen::Vector3d transform = translation;
```

Comment:
I would Eigen map the vector to a Mat3X array and apply the transform the the mapping.